package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "supertokens-auth-react",
-    "version": "0.39.1",
+    "version": "0.39.2",
```

Comment:
consider making this a breaking change?

Reply:
I'm fine with making this a breaking change.